package.json

```diff
@@ -139,6 +143,9 @@
       "mjs"
     ]
   },
+  "bin": {
+    "adminServe": "flow-node ./src/admin/localService/serve.js"
+  },
```

Comment:
curious, what's the reasoning for having this?

Reply:
This hasn't been completely worked out yet, but the idea is  that, rather than running a development build of the admin panel, we could release sourceCred with a parcel-managed production build of the admin panel that is served directly from this node service